scripts/logbook.py

```diff
@@ -38,76 +38,579 @@
-        "Quant-iT QC (Library Validation) 4.0" : {"dest_file" : ["CFX"], "instrument" : ["lims_instrument"], "details": [{"udf_Assay type" : "", "udf_Lot no: Quant-iT reagent kit" : ""}]},
-        "Quant-iT QC (RNA) 4.0" : {"dest_file" : ["CFX"], "instrument" : ["lims_instrument"], "details": [{"udf_Assay type" : "", "udf_Lot no: Quant-iT reagent kit" : ""}]},
-        "Qubit QC (DNA) 4.0" : {"dest_file" : ["Qubit"], "instrument" : ["lims_instrument"], "details": [{"udf_Assay" : "", "udf_Lot no: Qubit kit" : ""}]},
-        "Qubit QC (RNA) 4.0" : {"dest_file" : ["Qubit"], "instrument" : ["lims_instrument"], "details": [{"udf_Assay" : "", "udf_Lot no: Qubit kit" : ""}]},
-        "Qubit QC (Dilution Validation) 4.0" : {"dest_file" : ["Qubit"], "instrument" : ["lims_instrument"], "details": [{"udf_Assay" : "", "udf_Lot no: Qubit kit" : ""}]},
-        "Qubit QC (Library Validation) 4.0" : {"dest_file" : ["Qubit"], "instrument" : ["lims_instrument"], "details": [{"udf_Assay" : "", "udf_Lot no: Qubit kit" : ""}]},
-        "RAD-seq Library Indexing v1.0" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
-        "Ribosomal cDNA Depletion and 2nd Amplification (SMARTer Pico) 4.0" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
-        "RiboZero depletion" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
-        "Setup Workset/Plate" : {"dest_file" : ["Bravo"], "instrument" : ["lims_instrument"], "details" : [""]},
-        "Size Selection (Pippin)" : {"dest_file" : ["Pippin"], "instrument" : ["default"], "details": [{"udf_Type: Gel Cassette" : "", "udf_Lot no: Gel Cassette" : "", "udf_Type: Marker" : "", "udf_Lot no: Marker" : "", "udf_Lot no: Electrophoresis Buffer" : ""}]},
-        "Size Selection (Robocut)" : {"dest_file" : ["Bravo"], "instrument" : ["lims_instrument"], "details" : [""]},
-        "Shear DNA (SS XT) 4.0" : {"dest_file" : ["Covaris","Bravo"], "instrument" : ["lims_instrument","udf_Instrument Used"], "details" : [{"udf_Lot no: Covaris tube" : ""},""]},
-        "ThruPlex library amplification" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
-        "ThruPlex template preparation and synthesis" : {"dest_file" : ["Bravo","PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]}
+        "Adapter ligation and reverse transcription (TruSeq small RNA) 1.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "Adapter Ligation and 1st Amplification (SMARTer Pico) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
```

Comment:
If this `dict` is purely just for reference i.e. you will not modify it later in the code, then we can avoid assigning this empty list and handle it downstream accordingly (see my suggested `get_details` alternative)

P.S: Sorry I dint think of it yesterday when we were discussing :sweat_smile: